README.md

```diff
@@ -1,50 +1,80 @@
-## MCP (mobile control panel) Standalone 
+# MCP (mobile control panel) Standalone 
 
 The MCP Standalone is PoC for a per Namespace service that helps developers create and integrate mobile applications on OpenShift.
 
 
```

Comment:
@philbrookes fyi, I've updated the main README with the contents of the WALKTHROUGH you added,
with some modifications for working with the OpenShift UI extension